packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -276,31 +367,40 @@ class SnackBar extends StatelessWidget {
             child: child,
           );
         },
-        child: snackbar,
-      ),
-    );
+        child: snackBar,
+      );
+    }
+
+    return ClipRect(child: snackBarTransition);
   }
 
   // API for Scaffold.addSnackBar():
 
-  /// Creates an animation controller useful for driving a snack bar's entrance and exit animation.
-  static AnimationController createAnimationController({ @required TickerProvider vsync }) {
+  /// Creates an animation controller useful for driving a snack bar's entrance
+  /// and exit animation.
+  static AnimationController createAnimationController({
+    @required TickerProvider vsync,
+  }) {
     return AnimationController(
       duration: _kSnackBarTransitionDuration,
       debugLabel: 'SnackBar',
       vsync: vsync,
     );
   }
 
-  /// Creates a copy of this snack bar but with the animation replaced with the given animation.
+  /// Creates a copy of this snack bar but with the animation replaced with
+  /// the given animation.
   ///
   /// If the original snack bar lacks a key, the newly created snack bar will
   /// use the given fallback key.
-  SnackBar withAnimation(Animation<double> newAnimation, { Key fallbackKey }) {
+  SnackBar withAnimation(Animation<double> newAnimation, {Key fallbackKey}) {
```

Comment:
Doublecheck that this is the correct style for flutter/flutter

Reply:
Done, I put the spaces back